python/kfserving/kfserving/api/kf_serving_watch.py

```diff
@@ -21,21 +21,28 @@
 from ..utils import utils
 
 
-def watch(name=None, namespace=None, timeout_seconds=600):
+def watch(name=None, namespace=None, timeout_seconds=600, version=constants.KFSERVING_VERSION):
     """Watch the created or patched InferenceService in the specified namespace"""
 
     if namespace is None:
         namespace = utils.get_default_target_namespace()
 
-    tbl = TableLogger(
-        columns='NAME,READY,DEFAULT_TRAFFIC,CANARY_TRAFFIC,URL',
-        colwidth={'NAME': 20, 'READY':10, 'DEFAULT_TRAFFIC':15, 'CANARY_TRAFFIC':15, 'URL': 50},
-        border=False)
+    if version == 'v1alpha2':
+        tbl = TableLogger(
+            columns='NAME,READY,DEFAULT_TRAFFIC,CANARY_TRAFFIC,URL',
+            colwidth={'NAME': 20, 'READY': 10, 'DEFAULT_TRAFFIC': 15,
+                      'CANARY_TRAFFIC': 15, 'URL': 50},
+            border=False)
+    else:
+        tbl = TableLogger(
+            columns='NAME,READY,CANARY_TRAFFIC,URL',
```

Comment:
Canary is actually per component level in v1beta1, we can call `PREDICTOR_CANARY_TRAFFIC`

Reply:
Done.